docs/docs/guide/Model.md

```diff
@@ -165,7 +165,7 @@ You can also pass in an object for the optional `settings` parameter that is an
 | Name | Description | Type | Default |
 |------|-------------|------|---------|
 | return | What the function should return. Can be `document`, or `request`. In the event this is set to `request` the request Dynamoose will make to DynamoDB will be returned, and no request to DynamoDB will be made. If this is `request`, the function will not be async anymore. | String | `document` |
-| attributes | What document attributes should be retrieved & returned. This will use the underlying `ProjectionExpression` DynamoDB option to ensure only the attributes you request will be sent over the wire. If this value is `undefined`, then all attributes will be returned. | [String] | undefined |
+| attributes | What document attributes should be retrieved & returned. This will use the underlying `ProjectionExpression` DynamoDB option to ensure only the attributes you request will be sent over the wire. It also use the underlaying `ExpressionAttributeNames` DynamoDB option to avoid errors with attributes names that conflicts with a DynamoDB reserved word. If this value is `undefined`, then all attributes will be returned. | [String] | undefined |
```

Comment:
```suggestion
| attributes | What document attributes should be retrieved & returned. This will use the underlying `ProjectionExpression` DynamoDB option to ensure only the attributes you request will be sent over the wire. If this value is `undefined`, then all attributes will be returned. | [String] | undefined |
```

This is a bug fix. I don't think we need documentation changes.

Reply:
Got it.